src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/VsTestConsoleProcessManager.cs

```diff
@@ -220,15 +220,24 @@ private string[] BuildArguments(ConsoleParameters parameters)
 
             if (isNetCoreRunner)
             {
-                args.Insert(0, vstestConsolePath);
+                args.Insert(0, GetEscapeSequencedPath(vstestConsolePath));
             }
 
             return args.ToArray();
         }
 
         private string GetConsoleRunner()
+            => isNetCoreRunner ? ( string.IsNullOrEmpty(this.dotnetExePath) ? new DotnetHostHelper().GetDotnetPath() : this.dotnetExePath) : GetEscapeSequencedPath(this.vstestConsolePath);
+
+        private string GetEscapeSequencedPath(string path)
```

Comment:
I never heard any path to be referred to as "escape sequenced". I would call this quoted path.

Reply:
oh looks like we have a few references to those words in this repo already: https://github.com/microsoft/vstest/search?q=Escape+sequence

Reply:
They refer to "escape sequence" which is well established, and means prepending special characters with escape character. Your method just adds quotes. Quotes would sometimes be escaped, but are themselves not escape sequence. 

I am not trying to make this unnecessarily difficult. But if Sanan thinks this is good, let's merge it then.